examples/panel-basic/src/components/SimplePanel/SimplePanel.tsx

```diff
@@ -35,13 +35,13 @@ export function SimplePanel({
         {(config, alignedDataFrame) => {
           return (
             <>
-              <TooltipPlugin
+              <TooltipPlugin 
```

Comment:
not sure if you added them or your editor but all this whitespaces in this and all the other files at the end of the line are not necessary. not a big deal and won't break anything, but not required.

Reply:
didn't add intentionally, I'll be on the look out for this sort of thing in the future.